python/containers/fetch-device-secrets.sh

```diff
@@ -10,5 +10,6 @@ echo Fetching secrets
 get-secret THIEF_DEVICE_PROVISIONING_HOST THIEF-DEVICE-PROVISIONING-HOST
 get-secret THIEF_DEVICE_ID_SCOPE THIEF-DEVICE-ID-SCOPE
 get-secret THIEF_DEVICE_GROUP_SYMMETRIC_KEY THIEF-DEVICE-GROUP-SYMMETRIC-KEY
+get-secret THIEF_AI_CONNECTION_STRING THIEF-AI-CONNECTION-STRING
```

Comment:
AI lol

Reply:
AI - Application Insights.  I think maybe I need to rename this variable 

---
In reply to: [501948558](https://github.com/Azure/iot-sdk-longhaul/pull/13#discussion_r501948558) [](ancestors = 501948558)